snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/ecl/SnomedEclEvaluationRequest.java

```diff
@@ -504,22 +496,37 @@ private static Expression executeDescriptionSearch(BranchContext context, Expres
 		}
 		
 		final RevisionSearcher searcher = context.service(RevisionSearcher.class);
-		try {
-			
-			final Query<String> descriptionIndexQuery = Query.select(String.class)
-				.from(SnomedDescriptionIndexEntry.class)
-				.fields(SnomedDescriptionIndexEntry.Fields.CONCEPT_ID)
-				.where(descriptionExpression)
-				.limit(Integer.MAX_VALUE)
-				.build();
 			
-			final Hits<String> descriptionHits = searcher.search(descriptionIndexQuery);
-			final Set<String> conceptIds = Set.copyOf(descriptionHits.getHits());
-			return SnomedDocument.Expressions.ids(conceptIds);
-
-		} catch (IOException e) {
-			throw new SnowowlRuntimeException(e);
```

Comment:
The convenience method `Query#search` is throwing `IndexException` in such cases; hopefully there aren't any `catch` blocks on the "way up" that are now skipping over the instance.

Reply:
That is an error as well, and it should be reported as usual, this was unnecessarily just converted it to a runtime exception.